specification/repository.md

```diff
@@ -52,10 +52,8 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
 
 ### GitHub Actions
 
-- MUST use [GitHub
-  secrets](https://docs.github.com/en/actions/reference/encrypted-secrets) to
-  store sensitive data (auth tokens, passwords) and limit their usage to only
-  required places
+- MUST NOT use [GitHub
+  secrets](https://docs.github.com/en/actions/reference/encrypted-secrets)
```

Comment:
Does `GITHUB_TOKEN` count as a secret?
If you run tests using GHCR you *have* to use that

Reply:
Good comment.
For sure not (it is not explicitly set by us). I will add a remark that it does not count

Reply:
PTAL https://github.com/signalfx/gdi-specification/pull/186/commits/97830548256438eec54a815cfba27a3f57bfaf86